flang/include/flang/Parser/dump-parse-tree.h

```diff
@@ -206,6 +206,7 @@ class ParseTreeDumper {
   NODE(CompilerDirective, IgnoreTKR)
   NODE(CompilerDirective, LoopCount)
   NODE(CompilerDirective, AssumeAligned)
+  NODE(CompilerDirective, VectorAlways)
```

Comment:
Nit: Was this (the list of CompilerDirectives) previously in sorted order?

Reply:
It's not sorted; AssumeAligned already comes after LoopCount. Not sure what the thought process on the ordering is, I just added this after AssumeAligned because it is the most similar to that.

Reply:
That's probably my fault for not realizing it's (meant to be) sorted, when I added `AssumeAligned`.

Feel free to fix that at the same time.